platform/pulpcore/app/views/status.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from pulpcore.app.serializers.task import WorkerSerializer
+import status
```

Comment:
I'm surprised this worked for you. It should be a fully-qualified import, I think: ``from pulpcore.app import status``